src/content/en/updates/2017/04/nic58.md

```diff
@@ -0,0 +1,119 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: What's new in Chrome 58 for developers? IndexedDB 2.0, display fullscreen and new options for sandboxed iframes.
+
+{# wf_published_on: 2017-04-19 #}
+{# wf_updated_on: 2017-04-19 #}
+{# wf_featured_image: /web/updates/images/generic/new-in-chrome.png #}
+{# wf_tags: chrome58,new-in-chrome,iframe,sandbox,indexeddb,progressive-web-apps,fullscreen #}
+{# wf_featured_snippet: With Chrome 58, Progressive Web Apps are more immersive with display: fullscreen. IndexedDB 2.0 is now supported and sandboxed iFrames get more options. Pete LePage has all the details and how you can use these new developer features in Chrome 58! #}
+
+# New in Chrome 58 {: .page-title }
+
+{% include "web/_shared/contributors/petelepage.html" %}
+
+<div class="clearfix"></div>
+
+<div class="video-wrapper">
+  <iframe class="devsite-embedded-youtube-video" data-video-id="TODO"
+          data-autohide="1" data-showinfo="0" frameborder="0" allowfullscreen>
+  </iframe>
+</div>
+
+* The [IndexedDB 2.0](#idb2) standard is now fully supported in Chrome, and
+  features new schema management, bulk action methods, and more
+  standardized handling of failures.
+* Progressive Web Apps become more immersive with
+  [`display: fullscreen`](#fullscreen).
+* [`allow-top-navigation-by-user-activation`](#iframes) gives sandboxed
+  `iframes` new powers
+* And there’s plenty [more](#more)!
+
+<div class="clearfix"></div>
+
+I’m Pete LePage, let’s dive in and see what’s new for developers in Chrome 58! 
+
+<div class="clearfix"></div>
+
+## IndexedDB 2.0 {: #idb2 }
+
+The structure of your site’s database has large performance impacts, and can
+be difficult to change.
+[IndexedDB 2.0](https://hacks.mozilla.org/2016/10/whats-new-in-indexeddb-2-0/) 
+changes that.
+
+* `object` stores and `indexes` can now be renamed in-place after a
+  refactoring. 
```

Comment:
Not quite, it means that the object stores and indexes in an indexeddb created by a developer can be renamed in place. In the past, you couldn't rename an index, but now you can.